### DISCUSSION TITLE
Percentage of renewables / total consumption

### DISCUSSION THREAD

**Discussion by Raffaelemzz** [electricitymaps/electricitymaps-contrib#4946] [Ideas]
On the data visible on Electricity Maps and Terna, I noticed some important discrepancies.
For example, these are the data for January 11, 2023 at 03 AM:
![IMG_6434](https://user-images.githubusercontent.com/122230113/211835346-75d508a0-dfa8-4199-a468-852dfc7599f5.PNG)
![IMG_6433](https://user-images.githubusercontent.com/122230113/211835383-8cf597db-37ca-4020-8e57-3b26947e5568.PNG)
According to Terna, renewables covered 283.7% of electricity consumption in southern Italy.
Therefore I suggest adding another percentage of renewables to the site expressed as:
(production from renewables)/(total consumption) = (production from renewables)/(total production + imports - exports)

**Comment by VIKTORVAV99** [comment]
From the screenshot you shared it looks like TERNA don't take exchanges into account if you are going to compare it to Electricity Maps you should switch to production mode which only use a countrys own production values. Could you try that and see if it's more in line what you expect to see?

**Reply by Raffaelemzz** [reply]
On Electricity Maps in the "production mode" the percentage of renewables I think is equal to: (production from renewables) / (total production), therefore it is impossible to have a percentage higher than 100%.
While I believe that Terna calculates the percentage of renewables as: (production from renewables) / (total consumption).
In some areas (e.g. Southern Italy, France, Norway) the production of electricity is constantly higher than consumption, often the production from renewables exceeds internal consumption (therefore we have percentages higher than 100%).
Conversely, in areas where internal production is not sufficient to meet consumption (e.g. Northern Italy) the percentage of renewables displayed on Terna's app is lower than the percentage of renewables visible on Electricity Maps.

In my opinion, seeing the percentage of renewables as (production from renewables) / (total consumption) can be a new way to highlight some virtuous areas from the point of view of production capacity.

Eg:
Sicily January 11, 2023 at 3 AM
According to Electricity Maps, 49% of the energy produced comes from renewables (0 imports, so the percentage of renewables is 49% also in consumption mode)

Of the 2480 MW produced, 982 MW are exported.
The production of renewables is 1216 MW.
1216/2480= 49% (as visible on Electricity Maps)
1216/(2480-982)= 81% (approximately the value visible on the Terna app)

**Reply by VIKTORVAV99** [reply]
Ah yeah that's true.
I'll make a note of it and check in with the team when we are planning new features. However it might not be possible as we don't have consumption data for all zones.
